README.md

```diff
@@ -1 +1,47 @@
-# php-validator-livr-extra
\ No newline at end of file
+# PHP Validator LIVR Extra (uMotif Fork)
+Extension wrapper for [umotif-public/php-validator-livr](https://github.com/umotif-public/php-validator-livr).
+
+This library contains additional LIVR rules not found in the original uMotif package.
+
+## Installation
+Add the following to your `composer.json`:
+```json
+    {
+        "require": {
+            "umotif/livr-extra": "dev-master"
```

Comment:
We need to use a version here, or just have the documentation tell the reader to require the package via the CLI (without specifying a version) e.g `composer require umotif/php-livr-extra`